apache/zeppelin
# [gh-pages][ZEPPELIN-2209, 2110] Show 'INTERPRETER' type & Add pagination

### What is this PR for?
After #2037 merged, we can see `INTERPRETER` type in Helium menu. So for the consistency, it'll be better we can show`INTERPRETER` type in https://zeppelin.apache.org/helium_packages.html. So what I did in this PR is 
 - enabled to see `INTERPRETER` type 
 - added a pagination

### What type of PR is it?
Improvement

### Todos
* [x] - update overview sentences
(current overview sentences are only for npm package related description. But`INTERPRETER` type pkgs come from mvn central not npm registry. So need to update accordingly)

### What is the Jira issue?
* [ZEPPELIN-2110](https://issues.apache.org/jira/browse/ZEPPELIN-2110) : List community & 3rd party interpreter registered at Maven central repo in Helium menu 
* [ZEPPELIN-2209](https://issues.apache.org/jira/browse/ZEPPELIN-2209) : Add pagination

### How should this be tested?
1. To see `INTERPRETER` type pkg, replace [helium online registry uri](https://github.com/apache/zeppelin/blob/gh-pages/_includes/themes/zeppelin/default.html#L34) with `<script src="https://rawgit.com/AhyoungRyu/spark-notebook-example/master/helium-fake.js"></script>`
2. Run the website(`gh-pages`) locally as described in [here](https://github.com/apache/zeppelin/tree/gh-pages#run-website)

### Screenshots (if appropriate)
![record](https://cloud.githubusercontent.com/assets/10060731/23833560/254de8b0-078b-11e7-8083-4bd846f2fbfc.gif)


### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? yes need to update overview sentences as I said in TODO
